Doc/api/conf.py

```diff
@@ -92,7 +111,9 @@
 }
 
 # To avoid import errors.
-autodoc_mock_imports = ["MySQLdb", "Bio.Restriction.Restriction", "Bio.Alphabet"]
+autodoc_mock_imports = ["MySQLdb", "Bio.Restriction.Restriction"]
+if version > "1.77":
```

Comment:
That's string comparison - it will only work if we have two digit minor versions, "1.xx".

But like the 1.75 check earlier, it will work for now.